SingularityUI/app/templates/tasksTable/tasksActiveBody.hbs

```diff
@@ -4,15 +4,15 @@
             <thead>
                 <tr>
                     <th data-sort-attribute="taskId.id">
-                        Name
+                        Task ID
                     </th>
                     <th class="hidden-xs" data-sort-attribute="taskId.startedAt">
-                        Started
+                        Started At
                     </th>
                     <th class="hidden-xs" data-sort-attribute="host">
                         Host
                     </th>
-                    <th class="hidden-xs" data-sort-attribute="taskId.rackId">
+                    <th class="visible-lg" data-sort-attribute="taskId.rackId">
```

Comment:
At the smallest window sizes, the task ID and started at timestamp start running off the screen. But that happens regardless of what we're showing. hidden-xs seems to be ok - everything is able to display pretty much the same at the smaller sizes down to the xs cutoff. It might not be ok for a table with more columns though. Also, the actions icons (kill task and JSON) can fit without causing a change in the way the rest of the table looks even at the smallest sizes.